plugins/obs-filters/cmake/nvidia.cmake

```diff
@@ -0,0 +1,6 @@
+if(ENABLE_NVAFX)
+  target_sources(obs-filters PRIVATE noise-suppress-filter.c)
+  target_enable_feature(obs-filters LIBNVAFX_ENABLED HAS_NOISEREDUCTION)
```

Comment:
IIRC, `target_enable_feature` _requires_ a message, and the ARGN are optional things to pass to `target_compile_definitions`.
https://github.com/obsproject/obs-studio/blob/c521b23619f9938bc00de38d9ca15a3fe0061685/cmake/common/helpers_common.cmake#L67-L74

You either have to do:
```cmake
target_enable_feature(obs-filters "NVIDIA Audio FX migrations" LIBNVAFX_ENABLED HAS_NOISEREDUCTION)
```
or, if you don't want to call out the "feature" in the CMake configure printout:
```cmake
target_compile_definitions(obs-filters PRIVATE LIBNVAFX_ENABLED HAS_NOISEREDUCTION)
```

Reply:
ok, have put it to private then

Reply:
Just noting that the "PRIVATE" is not the distinction here. The main point is `target_enable_feature` vs. `target_compile_definitions`.